app/controllers/admin_controller.rb

```diff
@@ -8,7 +8,7 @@ class AdminController < ApplicationController
   before_action :load_questionnaires, only: %i[export preview_overview]
 
   def preview
-    @use_mentor_layout = @questionnaire.name.match?(/mentor/)
+    @use_mentor_layout = @questionnaire.name.match?(/mentor|evaluatieonderzoek/)
```

Comment:
blegh. Willen we dit misschien niet in de vragenlijst definitie opnemen? (niet nu, maar dat je eigenlijk per vragenlijst definitie kunt aangeven wat voor styling het moet gebruiken)

Reply:
Ja ik maak een ticket aan.